File: app/src/main/java/cu/axel/smartdock/db/DatabaseContract.java
package cu.axel.smartdock.db;

import android.provider.BaseColumns;

public class DatabaseContract {
    private DatabaseContract() {
    }
    public static class LaunchModesTable implements BaseColumns {
        public static final String TABLE_NAME = "table_launch_modes";
        public static final String COLUMN_PACKAGE_NAME = "package_name";
        public static final String COLUMN_LAUNCH_MODE = "launch_mode";
    }
}

File: app/src/main/java/cu/axel/smartdock/db/DatabaseContract.kt
package cu.axel.smartdock.db

import android.provider.BaseColumns

class DatabaseContract private constructor() {
    object LaunchModesTable : BaseColumns {
        const val TABLE_NAME = "table_launch_modes"
        const val COLUMN_PACKAGE_NAME = "package_name"
        const val COLUMN_LAUNCH_MODE = "launch_mode"
    }
}
